pkg/controller/jobs/job/job_webhook.go

```diff
@@ -104,7 +104,10 @@ func (w *JobWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log.V(5).Info("Applying defaults")
 
 	jobframework.ApplyDefaultLocalQueue(job.Object(), w.queues.DefaultLocalQueueExist)
-	copyRaySubmitterJobMetadata(ctx, job.Object(), w.client)
+	err := copyRaySubmitterJobMetadata(ctx, job.Object(), w.client)
+	if err != nil {
```

Comment:
nit, let's inline for readability `if err := copyRaySubmitterJobMetadata(ctx, job.Object(), w.client); err != nil `{